bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/skeleton/factory/package.scala

```diff
@@ -1,4 +1,38 @@
 package edu.gemini.phase2.skeleton
 
+import edu.gemini.model.p1.immutable.{Band, Observation, Proposal, TimeAmount}
+
+import scalaz._
+import Scalaz._
+
 package object factory {
```

Comment:
All of the methods you add here take `Proposal` as a single argument.  You could also bedazzle them into existence with an `implicit class` that takes a `Proposal` in its constructor.  No big deal but it gives you nicer syntax I think.